ops/solana/solana.go

```diff
@@ -64,39 +74,60 @@ func New(ctx *pulumi.Context) (Deployer, error) {
 	}
 
 	// Check gauntlet works
-	cwd, _ = os.Getwd()
-	gauntletBin := filepath.Join(cwd, "./bin/gauntlet-") + version
+	os.Chdir(cwd) // move back into ops folder
+	gauntletBin := filepath.Join(cwd, "../gauntlet/bin/gauntlet-") + version
 	gauntlet, err := relayUtils.NewGauntlet(gauntletBin)
 
 	if err != nil {
 		return Deployer{}, err
 	}
 
 	return Deployer{
-		gauntlet:  gauntlet,
-		network:   "local",
-		Contracts: make(map[int]string),
-		States:    make(map[int]string),
+		gauntlet: gauntlet,
+		network:  "local",
+		Account:  make(map[int]string),
 	}, nil
 }
 
 func (d *Deployer) Load() error {
+	// TODO: remove this - temporarily needed as artifacts are read directly from the root directory
+	// won't be needed once it reads from release artifacts?
+	cwd, _ := os.Getwd()
+	os.Chdir(filepath.Join(cwd, "../gauntlet")) // go from ops folder to gauntlet folder
```

Comment:
@RodrigoAD is this ready to be removed?

But also what if we have local contracts in development, and actually want to read the latest local artifacts?

Reply:
i think this is worth tackling as a part of the makefile since, like you pointed out, the artifacts will be different for local development